src/rule.hpp

```diff
@@ -42,23 +42,8 @@ class rule {
     rule(const rule &) = delete;
     rule &operator=(const rule &) = delete;
 
-    rule(rule &&rhs) noexcept
-        : enabled_(rhs.enabled_), source_(rhs.source_), id_(std::move(rhs.id_)),
-          name_(std::move(rhs.name_)), tags_(std::move(rhs.tags_)), expr_(std::move(rhs.expr_)),
-          actions_(std::move(rhs.actions_))
-    {}
-
-    rule &operator=(rule &&rhs) noexcept
-    {
-        enabled_ = rhs.enabled_;
-        source_ = rhs.source_;
-        id_ = std::move(rhs.id_);
-        name_ = std::move(rhs.name_);
-        tags_ = std::move(rhs.tags_);
-        expr_ = std::move(rhs.expr_);
-        actions_ = std::move(rhs.actions_);
-        return *this;
-    }
+    rule(rule &&rhs) noexcept = default;
+    rule &operator=(rule &&rhs) noexcept = default;
```

Comment:
No really a good practice to specify `noexcept` in defaulted constructors. This is because `noexcept` is [inferred](https://en.cppreference.com/w/cpp/language/noexcept_spec) (provided that no base class constructor, or default member initialization or some subexpression like default argument potentially throws; but if any of those is the case, then it probably shouldn't be marked `noexcept` anyway)

Reply:
Yeah you're right, I'll change it.